Fix failures to configure RAID with newish dracclient

priteau opened the pull request:
The failure was due to the deprecated `raid_state` attribute being removed.

priteau committed:
Add missing space to error message

priteau committed:
Fix failures to configure RAID with newish dracclient

The "Ensure that DRAC configuration is applied" task was failing with
the following error:

    FAILED! => {"changed": false, "msg": "Failed to configure DRAC: AttributeError(\"'PhysicalDisk' object has no attribute 'raid_state'\",)"}

The raid_state attribute was deprecated back in 2016 [1]. It was then
removed in release 2.0.0, released in November 2018.

[1] https://opendev.org/openstack/python-dracclient/commit/9ff7c31
[2] https://opendev.org/openstack/python-dracclient/commit/3cb32a9

priteau committed:
Clarify requirements for python-dracclient

markgoddard reviewed (commented):
ansible-lint complaining about local_action.

priteau committed:
Replace use of local_action

This fixes ansible-lint errors due to rule E504:

    Do not use local_action, use delegate_to: localhost
